src/queueboard/dashboard.py

```diff
@@ -458,7 +458,7 @@ def code_link(s: str) -> str:
   <li><strong>find PRs by modified file</strong>: search for a file name and find all PRs in a list which modify this file. Searching for several files finds all PRs which modify all these files. (Caveat: we only track the first 100 files each PR changes, so this may yield incomplete results for cross-cutting PRs. Such PRs are rare, however.)</li>
   <li><strong>find your PRs</strong>: searching for your user name returns all PRs you reviewed or commented on</li>
   <li><strong>search PR description</strong>: searching also searches a PR description, and the list of all users who ever commented on this PR. To find all PRs from the sphere-eversion project, searching for "sphere-eversion" (or "sphere eversion") should do the trick.</li>
-  <li><strong>configuration via URL</strong>: you can configure the initial sorting, search terms and number of entries per page by changing the URL you're visiting. Three short examples:
+  <li><strong>configuration via URL</strong>: if you change table settings (sorting, search terms, and / or number of entries per page), those settings will appear as query parameters in the URL and automatically applied to all tables when you visit that URL again. Three short examples demonstrating the URL format:
```

Comment:
```suggestion
  <li><strong>configuration via URL</strong>: if you change table settings (sorting, search terms, and / or number of entries per page), those settings will appear as query parameters in the URL and automatically will be applied to all tables when you visit that URL again. Three short examples demonstrating the URL format:
```

Reply:
```suggestion
  <li><strong>configuration via URL</strong>: if you change table settings (sorting, search terms, and / or number of entries per page), those settings will appear as query parameters in the URL and will be automatically applied to all tables when you visit that URL again. You can also use the URL fragment (the portion after the `#`) to focus on a particular table; this can also be set in the URL by clicking on the name of the table on the page.
  Three short examples demonstrating the URL format:
```